app/presenters/candidate_interface/application_form_presenter.rb

```diff
@@ -156,7 +156,7 @@ def volunteering_added?
     end
 
     def all_referees_provided_by_candidate?
-      @application_form.application_references.count == ApplicationForm::MINIMUM_COMPLETE_REFERENCES
+      @application_form.application_references.count >= ApplicationForm::MINIMUM_COMPLETE_REFERENCES
```

Comment:
It might make sense to make a similar change in the `ReceiveReference` service to be on the safe side, though I don't think it would get called 'out of order'. It gets called when the candidate submits but that's only for sandbox and wouldn't cause a problem there anyway.

Reply:
Updated the comment so that it's clearer we need to only progress applications when the number of received applications change from 1 to 2.